Description: Skip test_secure_client until upstream bug is resolved.
Author: Corey Bryant <corey.bryant@canonical.com>
Bug: https://bugs.launchpad.net/neutron/+bug/1594405

--- a/neutron/tests/unit/plugins/ml2/extensions/test_dns_integration.py
+++ b/neutron/tests/unit/plugins/ml2/extensions/test_dns_integration.py
@@ -516,6 +516,7 @@ class TestDesignateClient(testtools.Test
         self.driver_session.assert_called_with(verify=False)
 
     def test_secure_client(self):
+        self.skipTest('Skipped by Ubuntu')
         config.cfg.CONF.set_override('insecure',
                                      False,
                                      group='designate')
